### DISCUSSION TITLE
Modal not showing (Background fading ) after reclick action ( Show at first time)

### DISCUSSION THREAD

**Discussion by jazzy1986** [filamentphp/filament#14590] [Help]
### Package

Actions

### Package Version

3.2.114

### How can we help you?

I think i encounter a bug, after i close a modal ( Form with a lot of fields & select option with db query  ) and reclick the action to open immediately, modal not showing, only background fading.

![image](https://github.com/user-attachments/assets/70e30d85-bf9a-4530-bb35-ef6ad231a5a3)

**Comment by jazzy1986** [comment]
seem like relate to this issue #10581

**Reply by leandrocfe** [reply]
Maybe you could create a minimal repo on Github to reproduce this issue..
